src/style.css

```diff
@@ -0,0 +1,41 @@
+.card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.card__title {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__link {
+  display: block;
+  line-height: 40px;
+  color: #fff;
+  font-weight: 700;
+  font-size: 14px;
+}
+
+.card__buy .card__link:hover {
```

Comment:
And here shoould be .card__link--hover:hover {....} 
![image](https://user-images.githubusercontent.com/88103621/234006532-e7d91b00-ffd7-46b4-b9fb-a316b69eb10f.png)

Reply:
fixed